lib/workers/repository/update/pr/code-owners.ts

```diff
@@ -5,9 +5,20 @@ import { readLocalFile } from '../../../../util/fs';
 import { getBranchFiles } from '../../../../util/git';
 import { newlineRegex, regEx } from '../../../../util/regex';
 
+type UserScore = {
```

Comment:
use interface

Reply:
Solved in 1405529e47463d85a160e801d4993bd3372c1f80